packages/embed/package.json

```diff
@@ -32,7 +32,8 @@
     "form-napper": "^2.0.0"
   },
   "devDependencies": {
-    "@types/applepayjs": "^3.0.2"
+    "@types/applepayjs": "^3.0.2",
+    "eslint-plugin-import": "^2.26.0"
```

Comment:
do things work without this? It's already defined in the main `package.json`, in theory it should pick it up from the root

Reply:
It seems to work fine.
Done.